inc/scroom/gtk-helpers.hh

```diff
@@ -42,9 +42,9 @@ namespace Scroom
 
     void useRecursiveGdkLock();
 
-    inline GdkRectangle createGdkRectangle(int x, int y, int width, int height)
+    inline cairo_rectangle_int_t createGdkRectangle(int x, int y, int width, int height)
```

Comment:
We should probably rename that method now. Perhaps `createCairoIntRectangle()` ?